client/ayon_core/tools/publisher/publish_report_viewer/delegates.py

```diff
@@ -118,8 +127,8 @@ def _get_minus_pixmap(cls, size):
 
     @classmethod
     def _get_icon_color(cls, name):
-        if name == "error":
-            return QtGui.QColor(colors["error"])
+        if name in ["error", "warning"]:
+            return QtGui.QColor(colors[name])
         return QtGui.QColor(QtCore.Qt.white)
```

Comment:
You can be a bit more abstract here.
```suggestion
        return QtGui.QColor(colors.get(name, QtCore.Qt.white))
```

Reply:
done